packages/react-microlink/src/components/Card/CardText.js

```diff
@@ -3,13 +3,15 @@ import styled from 'styled-components'
 import NanoClamp from 'nanoclamp'
 
 const Clamp = ({children, className, lines}) => (
-  <NanoClamp className={className} lines={lines} text={children} is='p' />
+  <NanoClamp className={className} lines={lines} text={children} is="p" />
 )
 
 const CardText = styled(Clamp)`
-  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
-  color: #181919;
-  margin: 0;
+  &&& {
```

Comment:
is this a valid syntax? what it means?

Reply:
Yeah, it's _a thing_ haha.

In scss `&` references "itself", so chaining `&`, chains the selector.

```
.title {
  &&& {
    ...
  }
}

===

.title.title.title { ... }
```

Basically a non-invasive way of doing `!important`, cause you can overwrite these styles with a stronger selector (without needing `!important`).

Note, this doesn't work for selectors that can't be chained, ie. `h1` => `h1h1h1` wouldn't work

It's mentioned a bit in styled-components [docs](https://www.styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity)

Reply:
nice!